sympy/simplify/combsimp.py

```diff
@@ -217,6 +219,232 @@ def explicate(p):
                 denom_gammas.extend(l)
             elif isg is False:
                 denom_others.extend(l)
+        # =========== level >= 2 work: factor absorbtion =========
+        if level >= 2:
```

Comment:
It looks like this is a part of `combsimp`. Only `gammasimp` should be dealing with `gamma`. There could be something similar in `combsimp` but handling only factorials (possibly also rising and falling), not gammas.